charts/datadog/values.schema.json

```diff
@@ -0,0 +1,300 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "title": "Values",
+  "type": "object",
+  "properties": {
+    "datadog": {
+      "type": "object",
+      "properties": {
+        "apm": {
+          "type": "object",
+          "properties": {
+            "instrumentation": {
+              "type": "object",
+              "properties": {
+                "enabled": {
+                  "type": "boolean"
+                },
+                "enabledNamespaces": {
```

Comment:
Note: For things that are `[]string` we can have `$defs` in place to make this a bit more compact.

```json
{ 
 "...": "...",
 "properties": {
   "enabledNamespaces": { "$ref": "#/$defs/stringArray" },
   "disabledNamespaces": { "$ref": "#/$defs/stringArray" },
 },
 "$defs": {
   "stringArray": { "type": "array", "items": { "type": "string" } }, 
 }
```

This might be more useful for `matchExpressions` and `matchLabels` for the selectors :) 

https://json-schema.org/understanding-json-schema/structuring#defs